zingo-testutils/src/chain_generic_tests.rs

```diff
@@ -89,7 +89,7 @@ pub mod fixtures {
         let expected_fee = MARGINAL_FEE.into_u64()
             * match pooltype {
                 Transparent => 3,
-                Shielded(Sapling) => 4,
+                Shielded(Sapling) => 4, // but according to my reading of https://zips.z.cash/zip-0317, this should be 3
                 Shielded(Orchard) => 2,
             };
```

Comment:
this expected fee calculator cannot return the correct fee in this state, lets discuss at codeshare but i think we should have a dedicated PR that adds an expected fee calculator to zingo-testutils

Reply:
Out of date...

Reply:
This instance of expected fee is correct in this micro context, which is good enough for this test for now